frontend/src/app/Settings/Users/ManageUsersContainer.tsx

```diff
@@ -68,6 +70,35 @@ const RESET_USER_PASSWORD = gql`
   }
 `;
 
+const UPDATE_USER_NAME = gql`
+  mutation UpdateUserName(
+    $id: ID!
+    $firstName: String
+    $middleName: String
+    $lastName: String!
+    $suffix: String
+  ) {
+    updateUser(
+      id: $id
+      name: null
+      firstName: $firstName
+      middleName: $middleName
+      lastName: $lastName
+      suffix: $suffix
+    ) {
+      id
+    }
+  }
+`;
+
+const UPDATE_USER_EMAIL = gql`
```

Comment:
You should be able to use the generated mutation from https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/generated/graphql.tsx here instead of manually defining it. It auto-generates the new methods and you can just import the mutation methods like any other - [`useResendActivationEmailMutation`](https://github.com/CDCgov/prime-simplereport/blob/45bba17db190523af920bc4985666fe938b4ece6/frontend/src/app/Settings/Users/ManageUsersContainer.tsx#L9) is written this way. Using the generated methods is much nicer, we just haven't gotten around to converting all the old methods yet 🙂

Reply:
That answers something I was confused about! I like it.